laws/src/main/scala/cats/laws/DecidableLaws.scala

```diff
@@ -0,0 +1,45 @@
+package cats
```

Comment:
Run `headerCreateAll` to add the headers.

Reply:
Nice, thank you!